packages/app/src/lib/linkify.ts

```diff
@@ -0,0 +1,24 @@
+import linkifyit from "linkify-it";
+// Adds common tlds (dns institute ranking + common atproto apps) to fuzzy detection. default list already includes country zones and:
+// * - biz|com|edu|gov|net|org|pro|web|xxx|aero|asia|coop|info|museum|name|shop|рф
+const TLDS = [
+  "xyz",
+  "io",
+  "biz",
+  "online",
+  "club",
```

Comment:
Better add "chat" here, for our own site! :)
 ```suggestion
   "club",
   "chat",
```